src/components/EmailForm.js

```diff
@@ -18,9 +23,43 @@ export default function EmailForm({ className }) {
           aria-label="Recipient's email"
         />
         <div className="input-group-append">
-          <button className="btn btn-outline-secondary" type="button">
+          <button
+            className="btn btn-outline-secondary"
+            type="button"
+            onClick={handleShow}
+          >
             Get Notified
           </button>
+
+          <div className="modal" id="myModal" tabIndex="-1" role="dialog">
+            <div className="modal-dialog" role="document">
+              <div className="modal-content">
+                <div className="modal-header">
+                  <h5 className="modal-title">Thank You!</h5>
+                  <button
+                    type="button"
+                    className="close"
+                    data-dismiss="modal"
+                    aria-label="Close"
+                  >
+                    <span aria-hidden="true">&times;</span>
+                  </button>
+                </div>
+                <div className="modal-body">
+                  <p>You will be notified when the app is launched.</p>
```

Comment:
This text "You will be notified when the app is launched" is center-aligned inside the dialog when I run it in the UI. Is this the design? 
If not, then could it be left-aligned (in line the title "Thank You!")?
I feel the alignment of "Thank you" and "You will be notified.." should be same (either both center aligned or both left-aligned).